### ISSUE TITLE
Duration format after 1 year

### ISSUE THREAD

**Issue by jbeaujour** [sni/Thruk#1263]
Hello

**Describe the bug**
When displaying column Duration with a value > 1 year , thruk display only days over 1Y 

**Thruk Version**
Thruk V3.02

**To Reproduce**
1. Go to hosts/Services/Problem, etc.  
2. Click onDuration to sort on max duration, sort is good
3. See error :

**Expected behavior**
Display of days or year + days.

**Screenshots**
![image](https://user-images.githubusercontent.com/14891591/223107887-f2ca911c-d986-478f-a47f-fe7277e91b2c.png)

**Comment by sni** [comment]
indeed, thanks for noticing and reporting this. Should be fixed in the next release.
